consul/src/main/scala/io/buoyant/consul/v1/HealthStatus.scala

```diff
@@ -0,0 +1,44 @@
+package io.buoyant.consul.v1
+
+import com.fasterxml.jackson.core.{JsonGenerator, JsonParser}
+import com.fasterxml.jackson.databind.{DeserializationContext, SerializerProvider}
+import io.buoyant.config.{ConfigDeserializer, ConfigSerializer}
+
+object HealthStatus extends Enumeration {
+  type HealthStatus = Value
+  val Passing, Warning, Critical, Maintenance = Value
+  def withNameSafe(name: String): Value =
+    values.find(_.toString.toLowerCase == name.toLowerCase()).getOrElse(Passing)
```

Comment:
rm parens from `name.toLowerCase()` here please – parens on a zero-argument function [should be used to indicate that the function is executed for side effects](https://docs.scala-lang.org/style/method-invocation.html#arity-0)